job-server/config/dse.conf

```diff
@@ -14,12 +14,12 @@ spark {
 
   jobserver {
     port = 8090
-    jar-store-rootdir = /tmp/jobserver/jars
+    jar-store-rootdir = ${HOME}/.spark-jobserver/jars
```

Comment:
Does this config has variable expansion? Will Spark Jobserver eventually replace ${HOME} with actual path?

Reply:
yes, it does